proto/sdv/databroker/v1/types.proto

```diff
@@ -174,20 +174,20 @@ message ValueRestriction {
 }
 
 message ValueRestrictionInt {
-  optional sint64 min            = 1;
-  optional sint64 max            = 2;
+  sint64 min            = 1;
+  sint64 max            = 2;
   repeated sint64 allowed_values = 3;
 }
 
 message ValueRestrictionUint {
-  optional uint64 min            = 1;
-  optional uint64 max            = 2;
+  uint64 min            = 1;
+  uint64 max            = 2;
   repeated uint64 allowed_values = 3;
 }
 
 message ValueRestrictionFloat {
-  optional double min = 1;
-  optional double max = 2;
+  double min = 1;
+  double max = 2;
```

Comment:
This doesn't really work as numeric types in proto3 default to 0 (which would be indistinguishable from setting a min or max of 0).

I think the whole `ValueRestriction` concept is unnecessarily complex when compared to the previous approach of one field per restriction:
```proto
message Metadata {
  int32 id               = 1;
  EntryType entry_type   = 2;
  string name            = 4;
  DataType data_type     = 5;
  ChangeType change_type = 6;  // CONTINUOUS or STATIC or ON_CHANGE
  string description     = 7;

  Allowed allowed = 10;
  Value min = 11;
  Value max = 12;
};
```

So I would suggest that we revert adding `ValueRestriction` to sdv.databroker.v1, and preferrably update kuksa.val.v2 to use a more straight forward one field per restriction approach.

Reply:
But with the current implementation (in this PR) we always set min/max, right. I.e. if not specified in VSS we use for instance `f64::MIN`, so they are always explicitly set, right?